brand/models/embrace_campaign.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError as JsonValidationError
+
+
+def validate_configuration(json_config):
+    configuration_schema = {
+        "type": "object",
+        "additionalProperties": True,
+        "required": ["email", "system_prompt", "user_prompt"],
+    }
+
+    try:
+        validate(instance=json_config, schema=configuration_schema)
+    except JsonValidationError as error:
+        raise ValidationError(error.message)
+
+    try:
+        validate_email(json_config["email"])
+    except:
```

Comment:
It's bad practice to bare except statement here. If there's an exception that's typically thrown then just test for that